README.md

```diff
@@ -66,11 +66,26 @@ npm install
 3. Install required mkdocs packages using pip3.
 ```
 pip3 install -r requirements.txt
+pip3 install mike
 ```
 
-4. Run below command to start mkdocs server.
-```
-mkdocs serve
+4. Run below commands to use `mike` to deploy multiple versions of docs.
+```sh
+# Initially delete everything from gh-pages branch to start creating version folders.
+mike delete --all 
+
+# If above command throws an error please run below mkdocs command and then run mike deploy
+mkdocs gh-deploy --force
+
+# This command will create a folder named as v13 in gh-pages branch.
+mike deploy --push --update-aliases v13 latest
+mike set-default v13
+
+# This command will create a folder named as v12.x in gh-pages branch.
+mike deploy --push --update-aliases v12.x
```

Comment:
This is to create initial setup in `gh-pages` which will create a folder named `v12.x` and place all the doc changes in it. So when we select on v12.x from dropdown it will fetch docs from this folder.

Reply:
Since we won't have v12 docs should this be removed?

Reply:
Yes this can be removed if there is no v12 docs.

Reply:
Yes, I think we should.  @tomlyn do you agree?

Reply:
Sure. I have removed that from Readme.